scene/resources/style_box.cpp

```diff
@@ -146,6 +146,29 @@ StyleBox::StyleBox() {
 	}
 }
 
+void StyleBoxMulti::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("get_style_boxes"), &StyleBoxMulti::get_style_boxes);
+	ClassDB::bind_method(D_METHOD("set_style_boxes", "style_boxes"), &StyleBoxMulti::set_style_boxes);
+	ADD_PROPERTY(PropertyInfo(Variant::ARRAY, "style_boxes", PROPERTY_HINT_ARRAY_TYPE, MAKE_RESOURCE_TYPE_HINT("StyleBox")), "set_style_boxes", "get_style_boxes");
+}
+
+TypedArray<StyleBox> StyleBoxMulti::get_style_boxes() const {
+	return style_boxes;
+}
+
+void StyleBoxMulti::set_style_boxes(const TypedArray<StyleBox> &p_styleboxes) {
+	style_boxes = p_styleboxes;
```

Comment:
Setter probably should have a cyclic reference check to make sure one of the items is not `this` or another `StyleBoxMulti` with `this` as a child style, or it can get stuck it the infinite `draw` loop.

Reply:
Added a check for that in debug mode https://github.com/aaronfranke/godot/blob/style-box-multi/scene/resources/style_box.cpp#L159-L185

GitHub isn't showing the change on the PR for some reason. I could push again to see if that works but I will need to push again anyway for the docs after the version bump to 4.2 so I'll just wait.